Add scope labels to the contributing.md (#7346)
endless-sky/endless-sky: CONTRIBUTING.md

* **unconfirmed**: More information is needed to be sure this bug is really a bug.
* **unlikely**: An enhancement or other change that I consider lowest priority, too large or difficult, or not appropriate.
* **wontfix**: A change that I definitely do not think should be made.

> Some additional labels might be used to specify the scope of issues:

Reviewer: ```suggestion
```

There's no need to have these in a separate section. There's nothing separating "art" or "sound" from "content" or "enhancement" as an example.
I'd recommend putting them all in the same section.

> * **art**: Everything related to images, sprites, and graphics

Reviewer: ```suggestion
* **art**: A request for anything related to the addition or modification of images, sprites, and graphics.
```

> * **sound**: Everything to do with sound

Reviewer: ```suggestion
* **sound**: A request for the addition or modification of anything to do with sound effects, music or other audio.
```

> * **text - long**: Content that contains long blocks of text such as large missions and campaigns

Reviewer: ```suggestion
```
I don't really see any reason to add this to the contributing file, this specifically definitions of issue labels, there's not really any time we'd add text - long or text - short to an issue. Those labels are pretty much specifically for PRs. An issue would just use the more generic "content" label.

> * **text - short**: Content that contains short blocks of text such as jobs, small missions, descriptions, etc.

Reviewer: ```suggestion
```
ditto
